LOCALIZATION_CONTRIBUTORS

```diff
@@ -61,7 +61,7 @@ Mauricio Caldeira Baptista
 Natália Capra Ferrazzo
 André Galdino
 Marco Antônio Guimarães Auad Barroca
-
+Vinicius Barboza Ribeiro
```

Comment:
Please do still leave 2 blank lines at the end to separate from the next language group.
Thanks

Reply:
Sent a new edit request

Reply:
I've fixed this myself 13 minutes ago.
Thank you. It should be on its way to be merged soon now ;)

Reply:
Oh, my bad, did not catch that. I'm just getting the hang of this, but I'm eager to contribute. (:

Reply:
No problem. I'm happy to know that you want to join us and that you made the PR together with the crowdin request at the same time, following the instructions.